manifests/mod/security.pp

```diff
@@ -130,4 +130,35 @@
 
   unless $::operatingsystem == 'SLES' { apache::security::rule_link { $activated_rules: } }
 
+  file { "${::apache::params::docroot}/csp":
```

Comment:
I think the `$apache::params::docroot` can be overridden by the base class so is not a reliable location.